webcurator-harvest-agent-h3/src/main/resources/application.properties

```diff
@@ -32,6 +32,8 @@ spring.quartz.properties.org.quartz.threadPool.threadCount=3
 
 # name of the directory where the temporary harvest data is stored
 harvestAgent.baseHarvestDirectory=/usr/local/wct/harvest-agent
+# agent service url that the core knows about
+harvestAgent.baseUrl=http://localhost:${server.port}
 # agent protocol type that the core knows about
 harvestAgent.scheme=http
```

Comment:
If we have a base URL, do we also need a separate scheme?

Reply:
When fixing this PR, the v3.0/add-context-path PR wasn't merged at that time. So separated scheme, host and port variables were still here. These variables could be got rid of when v3.0/add-context-path is merged.